### DISCUSSION TITLE
[Feature Request] Windows 8.1 Start Screen

### DISCUSSION THREAD

**Discussion by XuanChen1234** [valinet/ExplorerPatcher#1688] [Ideas]
This is a great restoration project but the Windows 10 start screen would be SO much appreciated
![R-C](https://github.com/valinet/ExplorerPatcher/assets/140480110/fe361135-e99f-4b72-a868-29cff12d6cbf)

**Comment by pyrates999** [comment]
What are you asking for?

**Reply by vippium** [reply]
he is asking that he needs win8 start menu as a option like we have currently win10 and win11.

**Reply by pyrates999** [reply]
https://www.thewindowsclub.com/how-to-enable-windows-10-full-screen-start-menu-using-regedit-or-gpedit

**Comment by XuanChen1234** [comment]
Any more planned?

**Reply by Leokuma** [reply]
Unfortunately I don't think that's going to happen because I've never seen anyone else ask for that.
